### PR TITLE
SSY-3524, SSY-3591 Conversion Service improvements

### PR DESCRIPTION
**Pull request by nunomsfranco**
- SSY-3524 Conversion Service speed improvement 
    - file cleaning scheduler 
- SSY-3591 Logs readability improvement

### CODE REVIEW

File: docsbox/docs/tasks.py [hunk 1]
```diff
@@ -50,10 +44,10 @@ def process_convertion(path, options, meta):
             result = process_image_convertion(path, options, meta, current_task)
         else:
             return { "has_failed": True, "message": "Conversion for {0} is not supported".format(exportFormatType)}
-        if result and options["via_allowed_users"]:
+        if result and meta["save_in_via"]:
             r = save_file_on_via(app.config["MEDIA_PATH"] + current_task.id, result["mimeType"], options["via_allowed_users"])
             remove_file(app.config["MEDIA_PATH"] + current_task.id)
-            result['fileId'] = r.headers.get("Document-id")
+            result['fileId'] = r.headers.get("Document-id")
```
Comment (numsu) [inline docsbox/docs/tasks.py]: Remove redundant spaces

File: docsbox/docs/tasks.py [hunk 2]
```diff
@@ -64,16 +58,19 @@ def process_document_convertion(input_path, options, meta, current_task):
...
+        temp_path = input_path if check_file_content(input_path, output_path) == False else output_path
+        print("Temporary file is " + temp_path)
 
         force_ocr = 0
         while has_PDFA_XMP(temp_path) == False:
             if (force_ocr == 0):
                 script = app.config["OCRMYPDF"]["EXEC"] + [app.config["OCRMYPDF"]["OUT"] + "-" + options.get("output_pdf_version", None)]
             elif (force_ocr > 1):
                 raise Exception('It was not possible to convert file ' + meta["filename"] + ' to PDF/A.')
+            print("Running OCR My PDF as " + app.config["OCRMYPDF"]["FORCE"][force_ocr])
```
Comment (numsu) [inline docsbox/docs/tasks.py]: Add a print like this to the LO option as well if we want to log this kind of info

File: docsbox/docs/tasks.py [hunk 3]
```diff
@@ -64,16 +58,19 @@ def process_document_convertion(input_path, options, meta, current_task):
         script = app.config["GHOSTSCRIPT_EXEC"]
         if options.get("output_pdf_version", None) != "1":
             script[1] = script[1] + "=" + options.get("output_pdf_version", None)
+        print("Running ghostscript:")
```
Comment (numsu) [inline docsbox/docs/tasks.py]: I think we should rather log these to graylog

File: docsbox/docs/utils.py [hunk 4]
```diff
@@ -163,15 +170,16 @@ def remove_XMPMeta(file):
...
             if metadata is not None:
                 pdfa=app.config["PDFA"]
-                nodes = metadata.getNodesInNamespace("", pdfa["NAMESPACE"])
+                nodes = metadata.get_nodes_in_namespace("", pdfa["NAMESPACE"])
                 if get_pdfa_version(nodes) in pdfa["ACCEPTED_VERSIONS"]:
                     return True
             return False
-    except:
+    except Exception as e:
+        print(e)
```
Comment (numsu) [inline docsbox/docs/utils.py]: To graylog

**Review by numsu — APPROVED** [review]

File: docsbox/docs/tasks.py (hunk 2 above)
Comment (nunomsfranco) [reply docsbox/docs/tasks.py]: No i dont think we need since it already does it when running those services

**Pull request merged**